apps/cookbook/src/app/showcase/link-showcase/link-showcase.component.html

```diff
@@ -73,3 +73,18 @@ <h2>Custom trailing-icon</h2>
 <cookbook-example-viewer [html]="trailingTab.template" [css]="trailingTab.style">
   <cookbook-link-example-trailing-icon #trailingTab></cookbook-link-example-trailing-icon>
 </cookbook-example-viewer>
+
+<h2>Button Link</h2>
+<p>
+  If the UI calls for a link that performs an action (for instance, opening a modal) rather than
+  navigation, you can use a button with the
+  <code>kirby-link</code>
+  directive. This will give you a button that looks and feels like a link, avoiding the semantically
```

Comment:
Consider signalling a little more caution around usage - maybe something like:
```suggestion
  In some rare cases, specific UI-needs can warrant using a button that visually appears as a link, even though it actually performs an action (for instance, opening a modal) rather than navigating.
  In those cases, a button with the
  <code>kirby-link</code>
  directive can be used. This will give you a button that looks and feels like a link, avoiding the semantically
```